### PR TITLE
[16.0][MIG] mail_activity_done: Migration to 16.0

### PR DESCRIPTION
**Pull request by ignissak**

### CODE REVIEW

**Comment by ignissak** [comment]
I had to rewrite setup of the test because for some reason the related field `res_model` didn't get populated and returned None on the test. Previously only `res_model_id` was assigned to the record.

File: mail_activity_done/hooks.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    cr.execute(
+        """
+        UPDATE mail_activity
+        SET done = False
+        """
+    )
+
+
+def post_load_hook():
+    def _new_action_done(self, feedback=False, attachment_ids=None):
```
Comment (hbrunn) [inline mail_activity_done/hooks.py:35]: this monkey patch is potentially harmful and we don't need it. Would you mind cherry picking https://github.com/OCA/social/pull/1011 into your migration?

**Review by hbrunn — CHANGES_REQUESTED** [review]

File: mail_activity_done/models/res_users.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Copyright 2018-22 ForgeFlow <http://www.forgeflow.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+from odoo import api, fields, models, modules
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    @api.model
+    def systray_get_activities(self):
```
Comment (CasVissers-360ERP) [inline mail_activity_done/models/res_users.py:10]: I get a traceback:

`UncaughtPromiseError
Uncaught Promise > Identifying field "id" is lacking a value on ir.model with 'and' identifying mode
Error: Identifying field "id" is lacking a value on ir.model with 'and' identifying mode
    at ModelIndexAnd.findRecord (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10340:159)
    at ModelManager.findFromIdentifyingData (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10377:88)
    at ModelManager._insert (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10471:62)
    at ModelManager._preInsertIdentifyingFieldsFromData (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10490:20)
    at ModelManager.findFromIdentifyingData (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10377:6)
    at ModelManager._insert (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10471:62)
    at ModelField._insertOtherRecord (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10235:35)
    at ModelField._setRelationInsertAndReplace (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10239:71)
    at ModelField.parseAndExecuteCommands (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10219:40)
    at ModelManager._update (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10512:40)`

**Pull request closed without merging**